cms/templates/index.html

```diff
@@ -79,7 +83,10 @@ <h3 class="title">${_("Create a New Course")}</h3>
                   ## Translators: This is an example for the name of the organization sponsoring a course, seen when filling out the form to create a new course. The organization name cannot contain spaces.
                   ## Translators: "e.g. UniversityX or OrganizationX" is a placeholder displayed when user put no data into this field.
                   <input class="new-course-org" id="new-course-org" type="text" name="new-course-org" required placeholder="${_('e.g. UniversityX or OrganizationX')}" aria-describedby="tip-new-course-org tip-error-new-course-org" />
-                  <span class="tip" id="tip-new-course-org">${_("The name of the organization sponsoring the course.")}  <strong>${_("Note: The organization name is part of the course URL")}</strong> ${_("This cannot be changed, but you can set a different display name in Advanced Settings later.")}</span>
+                  <span class="tip" id="tip-new-course-org">${_("The name of the organization sponsoring the course. {link_start}Note: The organization name is part of the course URL.{link_end} This cannot be changed, but you can set a different display name in Advanced Settings later.").format(
+                      link_start=HTML('<strong>'),
```

Comment:
Make this `strong_start` and `strong_end` throughout, unless it really is a link.